add-link

```diff
@@ -3941,3 +3941,5 @@ https://www633321.com/vanila/
 https://xamo-82976.web.app/#
 https://yeicotjj.github.io/auditoria/continue.html
 https://yuppiiechef.com/account/ű
+https://www.devblog-dofus.com/fr/mmorpg/actualites/recompense/parchemins/succes.php
```

Comment:
Next time, you should not be needing to add a url, when you add the domain..

On the other hand, the url "source" for the phishing are ab most welcome in the MR/Issue description